ceph/ceph.py

```diff
@@ -44,10 +45,6 @@
     is_block_device,
     zap_disk,
     is_device_mounted)
-from utils import (
```

Comment:
I removed this import because it was importing a function from the ceph-mon hooks.  I copied the function straight into this ceph.py file